content/en/platform/corda/4.11/community/cordapp-constraint-migration.md

```diff
@@ -133,7 +133,7 @@ These instructions only apply to CorDapp Contract JARs (unless otherwise stated)
 Corda 4.5 requires some additional steps to consume and evolve pre-existing on-ledger **CZ whitelisted** constrained states:
 
 
-* As the original developer of the CorDapp, the first step is to sign the latest version of the JAR that was released (see [Building and installing a CorDapp](cordapp-build-systems.md)).
+* As the original developer of the CorDapp, the first step is to sign the latest version of the JAR that was released (see [Building and installing a CorDapp]({{< relref "cordapp-build-systems.md" >}})).
 The key used for signing will be used to sign all subsequent releases, so it should be stored appropriately. The JAR can be signed by multiple keys owned
 by different parties and it will be expressed as a `CompositeKey` in the `SignatureAttachmentConstraint` (See [API: Core types](api-core-types.md)).
```

Comment:
Add relref here?

Reply:
Done - also any other ones with punctuation such as colons in the link text.